### ISSUE TITLE
pulseaudio module doesn't parse html

### ISSUE THREAD

**Issue by michaelnew** [Alexays/Waybar#122]
I'm trying to set each module to output white text with a colored icon, and they all seem to work fine except for the `pulseaudio` module. 

In the `cpu` module, for example, this works great: `"format": "<span color=\"white\">{usage}%</span> "`

but in the `pulseaudio` module I've used something similar: `"format": "<span color=\"white\">{volume}%</span> {icon}"`. As you can see in the screenshot, it just ignores the fact that there's html it's supposed to be parsing.

![image](https://user-images.githubusercontent.com/522101/50068428-b244c500-017a-11e9-8930-3487272892b4.png)

As kind of a side-note, it would be nice to have the icon colors change for different states, which was something I was able to do in polybar. The battery icon turns when when below 15%, for example. Is there a good way to do that?

**Comment by Robinhuett** [comment]
For the battery module you can use format strings.
For example:
```
"format": "{usage}% <span color=\"green\"></span>",
"format-warning": "{usage}% <span color=\"red\"></span>",
"states": {
    "warning": 30
}
```

Ond regarding the pulseaudio problem, this part has to be changed:
https://github.com/Alexays/Waybar/blob/c7b0639f32febba2d7ff0746a3931e24ab96d997/src/modules/pulseaudio.cpp#L210-L212
I can make a pull when I'm back home

**Comment by Alexays** [comment]
Oh my bad, i forgot to do it in this module, it's fixed in 66ad2864c206e3887a208cfaf586c7684ec243a5

**Comment by michaelnew** [comment]
Fantastic, thank you!
